src/Tribe/Admin/Notice/Marketing.php

```diff
@@ -117,4 +144,23 @@ public function notice_gutenberg() {
 		return $notice;
 	}
 
+	/**
+	 * Announcement Notice for Virtual Events.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The notice html.
+	 */
+	public function virtual_events_notice() {
+		$link = sprintf(
+			'<a href="%1$s" target="_blank">%2$s</a>',
+			esc_url( 'http://m.tri.be/1aky' ),
+			esc_html__( 'Virtual Events for The Events Calendar', 'the-events-calendar' )
+		);
+
+		return sprintf(
+			_x( 'Just launched: %1s! Identify events as virtual events, Zoom integration, hidden livestream links and more.', 'Virtual Events Annoucement Banner', 'the-events-calendar' ),
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: fa072d915b3c4a0a3e6db61fa5f04787</sub>